README.md

```diff
@@ -241,22 +241,36 @@ releases.
 To the extent that it is feasible, the `nightly-testing` branch is
 kept up to date with Lean nightlies. The process for doing so is
 partially automated. The CI config file
-[update-nightly.yml](.github/workflows/update-nightly.yml) runs
-regularly. If it detects a newer nightly than the one in
+[`update-nightly.yml`](.github/workflows/update-nightly.yml) 
+[runs](https://github.com/leanprover/reference-manual/actions/workflows/update-nightly.yml)
+every six hours. If it detects a newer nightly than the one in
 `nightly-testing`'s `lean-toolchain` file, then it attempts to change
-the contents of the file to the latest nightly and build the HTML
+the contents of `lean-toolchain` to the latest nightly and build the HTML
 version of the manual. On success, it commits the result and pushes it
 to `nightly-testing`, adding the tag
-`nightly-testing-YYYY-MM-DD`. When the automated process fails, a
-human-created PR to `nightly-testing` is required. When a
-human-created adaptation PR is pushed to `nightly-testing`, the
-`nightly-testing-YYYY-MM-DD` tag is created by
-`tag-nightly-testing.yml`.
+`nightly-testing-YYYY-MM-DD`.
+
+When the automated process fails, a human-created PR to `nightly-testing` is
+required. 
+1. Create a branch `bump-YYYY-MM-DD` file off of the `nightly-testing`
+   branch
+2. Update the `lean-toolchain` to the latest available 
+   `leanprover/lean4:nightly-*` toolchain
+3. Make any changes to the manual necessary to build the manual on top of the
+   new nightly.
+After these changes are added back to `nightly-testing` via PR, the 
+[`tag-nightly-testing.yml`](.github/workflows/tag-nightly-testing.yml) action
+[runs](https://github.com/leanprover/reference-manual/actions/workflows/tag-nightly-testing.yml)
+automatically and applies the `nightly-testing-YYYY-MM-DD` tag.
 
 The [`merge-main-nightly.yml`](.github/workflows/merge-main-nightly.yml)
-workflow runs every six hours. It attempts to merge `main` into
-`nightly-testing` and generate HTML. If this succeeds, then the result
-is pushed to `nightly-testing`.
+workflow 
+[runs](https://github.com/leanprover/reference-manual/actions/workflows/update-nightly.yml)
+every six hours. It attempts to merge `main` into `nightly-testing` and 
+generate HTML. If this succeeds, then the result is pushed to 
+`nightly-testing`. If it fails, human intervention is required. (Hint: make 
+sure that the toolchain is up-to-date before trying to fix failures of merging
+main into nightly.)
```

Comment:
Why give this hint?

I think the two tasks serve different purposes, and it can be reasonable to integrate `main` before doing a version bump. It's more important to do the version bump, but it's not a prerequisite to merging `main`.

Reply:
This is precisely what tripped me up today: I did the merge update first and this was precisely the wrong thing to do. The `main` branch contained changes that made the manual compatible with the `4.27.0-rc1` release, and when i merged `main` into `nightly`, I kept *undoing* those changes in order to make the manual compatible with `nightly-2025-12-12`. 

This is a problem specifically when a release happens: nightly can get slightly *behind* master. I could make this hint conditional: "Hint: a release updates the toolchain on the `main` branch, and `nightly` can fall behind. When this happens, make sure to update the toolchain to the latest nightly before trying to fix failures from merging
main into nightly."

Reply:
That sounds like a good addition.